feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
+
+One more change in the feature1 branch
```

Comment:
Amazing line of code) (part of my course)

Reply:
thx ()